docs/pages/components/date-picker.md

```diff
@@ -22,18 +22,18 @@ This component mostly relies on the CSS of other components and has very little
 	<div class="fd-popover">
 		<div class="fd-popover__control">
 			<div class="fd-input-group fd-input-group--after">
-				<input type="text" value="" placeholder="Pick a date">
+				<input type="text" class="fd-input" value="" placeholder="Pick a date">
 				<span class="fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button">
 				<button class="fd-popover__control fd-button--light sap-icon--calendar" aria-controls="Itcgq828" aria-expanded="false" aria-haspopup="true" aria-hidden="true"></button>
 				</span>
 			</div>
 		</div>
-		<div class="fd-popover__body fd-popover__body--right fd-popover__body--no-arrow" aria-hidden="true" id="Itcgq828">
+		<div class="fd-popover__body fd-popover__body--left fd-popover__body--no-arrow" aria-hidden="true" id="Itcgq828">
```

Comment:
Is this right?  Is it really moving from right-aligned to left-aligned?

Reply:
According to the newest design sheet, it should be left-aligned.